data/en/createguid.json

```diff
@@ -10,5 +10,13 @@
 		"lucee": {"minimum_version": "", "notes": "", "docs": "http://docs.lucee.org/reference/functions/createguid.html"}
 	},
 	"links": [],
-	"examples": []
+	"examples": [
+		{
+			"title":"Creates a Globally Unique Identifier (GUID). Each GUID will be unique",
+			"description":"",
+			"code":"createGUID()",
```

Comment:
You could prove that it is always unique by something like `createGUID() neq createGUID()`.

Reply:
Lets do that as a second example (feel free to do that as another PR @CraicOverflow89 )... I think people may want to run that one just to see what the guid looks like.